mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule ExImageInfo.Mixfile do
       description:
         "ExImageInfo is an Elixir library to parse images (binaries) and get the dimensions (size), detected mime-type and overall validity for a set of image formats. It is the fastest and supports multiple formats.",
       version: "VERSION" |> File.read!() |> String.trim(),
-      elixir: "~> 1.3",
+      elixir: "~> 1.13",
```

Comment:
I wasn't sure about this bump, but I considered also this [comment]( https://github.com/Group4Layers/ex_image_info/pull/15#discussion_r1923845910) and the CI changes of this PR.